astropy/units/function/core.py

```diff
@@ -405,9 +412,31 @@ def to_string(self, format="generic"):
             # need to strip leading and trailing "$"
             self_str += rf"$\mathrm{{\left( {pu_str[1:-1]} \right)}}$"
         else:
-            self_str += f"({pu_str})"
+            pu_lines = pu_str.splitlines()
+            if len(pu_lines) == 1:
+                self_str += f"({pu_str})"
+            else:
+                # If the physical unit is formatted into a multiline
+                # string, the lines need to be adjusted so that the
+                # functional string is aligned with the fraction line
+                # (second one), and all other lines are indented
+                # accordingly.
+                l = len(self_str) + 1
```

Comment:
Maybe rename to make it visually less similar to 1? Or just put it straight into the f-string below.